### ISSUE TITLE
bug(www): /features/ tables are inaccessible

### ISSUE THREAD

**Issue by tesseralis** [gatsbyjs/gatsby#25672]
## Description

The tables featured in pages like [Gatsby vs Next.js](https://www.gatsbyjs.org/features/jamstack/gatsby-vs-nextjs/) are inaccessible to screenreader users. Among the issues:

* The tables are not [semantic `<table>` tags](https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/features/evaluation-table.js#L23). This means that table cells will not be read with proper context to screenreader users.
* None of the markers are picked up by screenreaders. They need some sr-only text or aria-labels to make sure that information is being read to screenreader users.
* The [legend table](https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/features/legend-table.js) does not use a `<table>` tag either

I just think it's ironic that a page that's supposed to advertise Gatsby's accessibility features is itself not accessible:

<img width="682" alt="Screenshot of a feature page showing Gatsby with high marks for accessibility" src="https://user-images.githubusercontent.com/1278991/87123075-edb25300-c23a-11ea-9fa6-f00d9b8022f2.png">

**Comment by jayblack388** [comment]
@tesseralis Any chance I can take a whirl at this for a first contribution? Seems like pretty straightforward changes.

**Comment by duncanjbain** [comment]
Hi @jayblack388 , I've been having a look at this also, have you made any progress? I think this issue might require a couple of PRs as the problems @tesseralis seem to be split across several components and pages, for example there are two components available to render the tables, with the pages that require tables using one or the other for rendering them.

**Comment by tesseralis** [comment]
@duncanjbain yup! I think each issue should preferably be worked on in its own PR, as they're pretty independent. Maybe you and @jayblack388 could tackle different portions of it?

**Comment by LekoArts** [comment]
I fixed most of the issues, only the button trigger and shown/hidden text needs some extra work which is being tracked in a ticket.
